powershell/models/TrieNode.ts

```diff
@@ -0,0 +1,40 @@
+export class TrieNode {
+  value: string;
+  private children: Map<string, TrieNode>;
+  isEnd: boolean;
+  constructor(value: string) {
+    this.value = value;
+    this.children = new Map<string, TrieNode>();
+    this.isEnd = false;
+  }
+
+  addChild(child: TrieNode): void
+  addChild(child: string): void
+  addChild(child: unknown): void {
```

Comment:
If addChild only accepts TrieNode or String as input, suggest to change it to addChild(child: TrieNode | string).

Reply:
fixed